src/components/ContentOpenWith/ContentOpenWith.js

```diff
@@ -229,21 +238,24 @@ class ContentOpenWith extends PureComponent<Props, State> {
                 this.executeIntegrationSuccessHandler,
                 this.executeIntegrationErrorHandler,
             );
+
+        this.executeId = appIntegrationId;
     };
 
     /**
      * Opens the integration in a new tab based on the API data
      *
      * @private
-     * @param {ExecuteAPI} executePostData - API response on how to open an executed integration
+     * @param {ExecuteAPI} executeData - API response on how to open an executed integration
 
      * @return {void}
      */
-    executeIntegrationSuccessHandler = (executePostData: ExecuteAPI): void => {
-        const { method, url } = executePostData;
+    executeIntegrationSuccessHandler = (executeData: ExecuteAPI): void => {
+        const { onExecute }: Props = this.props;
```

Comment:
Since this is only used once, you could use it inline: `this.props.onExecute(this.executeId);` Same goes for most of the other callback references in this file.